pkg/server/container_create.go

```diff
@@ -194,8 +196,16 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 
 	securityContext := config.GetLinux().GetSecurityContext()
 
+	processLabel, mountLabel, err := initSelinuxOpts(securityContext.GetSelinuxOptions())
```

Comment:
nit: Create a local variable for `securityContext.GetSelinuxOptions()`, so that we could make the next line shorter. :p

Reply:
Please add unit test for this. Basically, just set selinux options, and check whether `ProcessSelinuxLabel` and `LinuxMountLabel` are set correctly.

We don't need to add `SelinuxRelabel` for any mount, so that `Relabel` won't be actually run in the unit test.

Reply:
@Random-Liu  I have tried to add unit test for this, but the result of test depends on whether 
selinux have been enabled, see https://github.com/opencontainers/selinux/blob/4a2974bf1ee960774ffd517717f1f45325af0206/go-selinux/label/label_selinux.go#L28.

Reply:
@Random-Liu I have add the unit test with build tag `selinux`.